wormhole-connect/src/hooks/useFetchTokenPricesV2.ts

```diff
@@ -0,0 +1,69 @@
+import config from 'config';
+import { useEffect, useState } from 'react';
+
+const COINGECKO_URL = 'https://api.coingecko.com/';
+const COINGECKO_URL_PRO = 'https://pro-api.coingecko.com/';
+
+const useFetchTokenPricesV2 = (): {
+  prices: any;
```

Comment:
Can we (ideall) specify a different type or add a ts-ignore to suppress the linter error?